src/date/index.ts

```diff
@@ -27,3 +31,378 @@ export function getMonthAbbrByIndex(monthNumber: number): string {
 
   return monthNames[ monthNumber - 1 ];
 }
+
+
+/**
+ * This function converts date input coming from backend (generally in long format (ex. '2022-02-01T12:16:13'))
+ * to a specified format provided in 2nd paramter
+ * It converts all types of dates accepted by dayjs
+ *
+ * @param {Date} longDateValue - Date input to be converted to another format. This generally comes from backend and is
+ * in long format. For example '2022-02-01T12:16:13'
+ * @param {string} dateFormat - Format in which the date needs to be converted to.
```

Comment:
Can add link to the dayjs format method documentation. Which shows the list of the possible formats.

Reply:
Done